src/Button/Button.js

```diff
@@ -13,12 +16,50 @@ export const Button = ({ className, children, onClick, type, ref, ...props }) =>
   // Levers
   // ---------------------------------------------------------------------------
 
+  // Props.variant
+  if (props.variant && buttonVariants.includes(props.variant)) {
+    switch (props.variant) {
+      case 'primary':
+        levers.variant = classnames('su-bg-digital-red su-text-white su-border-2 su-border-digital-red su-border-solid hover:su-border-black focus:su-border-black')
+        break
+
+      case 'secondary':
+        levers.variant = classnames('su-bg-transparent hover:su-bg-transparent focus:su-bg-transparent su-text-digital-red hover:su-text-black focus:su-text-black su-border-2 su-border-digital-red su-border-solid hover:su-border-black focus:su-border-black')
+        break
+
+      case 'none':
+        levers.variant = classnames('su-bg-transparent hover:su-bg-transparent focus:su-bg-transparent')
+        break
+    }
+  }
+
+  // Props.size
+  if (props.size && buttonSizes.includes(props.size)) {
+    switch (props.size) {
+      case 'default':
+        levers.size = classnames('su-px-26 su-py-10 su-text-20')
+        break
+
+      case 'big':
+        levers.size = classnames('su-px-34 su-py-15 su-text-22')
+        break
+
+      case 'small':
+        levers.size = classnames('su-px-19 su-py-9 su-text-18')
+        break
+
+      case 'minimal':
+        levers.size = classnames('su-p-0')
+        break
+    }
+  }
+
   return (
     <button
-      className={classnames('su-button', levers.button, className)}
+      className={classnames('su-button', levers.variant, levers.size, className)}
```

Comment:
```suggestion
      className={classnames('su-button', ...levers, className)}
```

You should only have to include the object as classnames should be able to handle that.

Reply:
Hmm I tried this but it didn't seem to work